### PR TITLE
Add monitor mode

### PR DESCRIPTION
**Pull request by jeremy-israel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,10 @@ Datadome.configure do |config|
 
   # Add exclude matchers (optional)
   config.exclude_matchers << ->(host, path) { path =~ /\.(jpg|jpeg|png|gif)/i }
+  
+  # Enable monitoring mode (optional)
+  # Does not block incoming requests flagged as coming from a bot (useful for logging/monitoring)
+  config.monitoring_mode = true
```
Comment (misdoro) [inline README.md:50]: WDYT of calling it `monitor_mode` ?

File: lib/datadome/inquirer.rb [hunk 2]
```diff
@@ -26,6 +27,11 @@ def enriching
 
       headers.merge!(added_headers)
 
+      if @monitoring_mode
```
Comment (misdoro) [inline lib/datadome/inquirer.rb:30]: WDYT of adding a dedicated variable like `expose_headers`?

File: README.md (hunk 1 above)
Comment (jeremy-israel) [reply README.md]: ok for it

File: lib/datadome/inquirer.rb (hunk 2 above)
Comment (jeremy-israel) [reply lib/datadome/inquirer.rb]: good idea

**Pull request merged**